library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
@teryhill I tested it by making a function
```
<?php
function include_boostrap()
{?>
    <link rel="stylesheet" href="<?php echo $GLOBALS['standard_js_path'] ?>bootstrap-3-3-4/dist/css/bootstrap.min.css" type="text/css">
    <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-1.9.1.min.js"></script>
    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path'];?>/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>

<?php
}?>
```
Included this function in encounter form , tried changing some UI to Boostrap UI. Works fine there.